src/Command/Forums.js

```diff
@@ -0,0 +1,45 @@
+const Command = require('../Command');
+
+const messages = {
+    'errorBadKey': 'Hmm.. I couldn\'t find the game you were talking about.  Try again?',
+    'result': 'You can talk all about {game} here: <{link}>',
+    'help': 'You can use `!forums` to get a link to the forum in general or for a specific game.\nYou can use any of the following names: {names}'
+};
+
+module.exports = Command.extend({
+    commandName: 'forums',
+    processMessage: function (message, tokens) {
+        tokens.shift();
+        let key = this.getGameKey(tokens.join(' ').trim(), message.channel.id);
+
+        if (key === 'help') {
+            let list = Object.values(this.config.games.names)
+            .map(item => {
```

Comment:
This should be indented - i was momentarily confused b/c at a glance it looks like a fresh line of code